src/bindgen/parser.rs

```diff
@@ -238,6 +238,10 @@ impl<'a> Parser<'a> {
     }
 
     fn parse_mod(&mut self, pkg: &PackageRef, mod_path: &FilePath) -> Result<(), Error> {
+        self.parse_mod_depth(pkg, mod_path, 0)
```

Comment:
Can we just pass zero on the relevant callers and keep `parse_mod` as the name for all functions?

Reply:
I actually hesitated between the two options, fixed in 0d32d99